assets/js/player.js

```diff
@@ -112,6 +112,37 @@ function addCurrentTimeToURL(url) {
     return urlUsed;
 }
 
+/**
+ * Callback that updates the timestamp on all external links
+ */
+player.on('timeupdate', function () {
+    // Only update once every 5 seconds
+    if ((Math.ceil(player.currentTime()) % 5) != 0) return;
```

Comment:
If I'm not mistaken here, the links will pretty much never get updated. Since `timeupdate` fires every 15 - 250ms and the `player.currentTime()` returns the current time in seconds, the chances of reaching a perfect multiple of 5 would be quite low. 

--- 

On a semirelated note,

Why should the link only get updated every 5 seconds? The performance impact of increasing it to every second or even to each occurrence of the `timeupdate` event should be negligible.

Reply:
I'm not sure how often that event is fired. I couldn't find the info in VideoJS' docs nor code. As for the 5s delay, I feel like every second is too often. Plus, by going a few seconds back, It's easier to catch up, imo.

But if you think it's not useful, I can remove that.

Reply:
> I'm not sure how often that event is fired. I couldn't find the info in VideoJS' docs nor code. 

It fires every 15 - 250ms as seen here https://docs.videojs.com/docs/api/player.html#Eventstimeupdate

> But if you think it's not useful, I can remove that.

I mostly just want to copy YouTube's behavior which by default sets the timestamp as the current second in their share widget. You can decide if you'll like to keep it at updating every 5 seconds.

Reply:
I'm not sure how to cleanly update only once a second.
That'd be better if we don't update all the links every 15ms x)

Reply:
Ah, well lets keep it at 5 seconds for now then. A future PR can resolve this.